app/models/health/agency_performance.rb

```diff
@@ -324,6 +326,17 @@ def qa_signature_dates
         pluck(:patient_id).uniq
     end
 
+    private def with_annual_well_care_visit_in_12_months
+      # range is a 12 month period ending 3 months ago, because claims data is always 3 months behind
+      service_range = (Date.current - 15.months...Date.current - 3.months)
```

Comment:
I think we can simplify this to just being the last 12 months...

And then complicate the view, by determining the newest service date in the range, and reporting that as the end date for the calculation.